docs/csharp/tutorials/inheritance.md

```diff
@@ -0,0 +1,294 @@
+
+   For our example, we'll mark our derived class as `sealed`.
+
+The following example shows the source code for the `Publication` class, as well as a `PublicationType` enumeration that is returned by the `Publication.PublicationType` property. In addition to the members that it inherits from @System.Object, the `Publication` class defines the following unique members and member overrides:
+
+[!CODE [Inheritance](../../../samples/snippets/csharp/tutorials/inheritance/base-and-derived.cs#1)]
+
+- A constructor
+
+  Because the `Publication` class is `abstract`, it cannot be instantiated directly from code like the following:
+
+  ```cs
+  var publication = new Publication("Tiddlywinks for Experts", "Fun and Games",
+                                    PublicationType.Book);
+  ```
+
+  However, its class constructor can be called directly from derived class constructors, as the source code for the `Book` class shows.
+
+- Two publication-related properties
+
+  `Title` is a read-only @System.String property whose value is supplied by calling the `Publication` construction, which stores the value in a private field named `pubTitle`.
+
+  `Pages` is a read-write @System.Int32 property that indicates how many total pages the publication has. The value is stored in a private field named `totalPages`. It must be a positive number or an @System.ArgumentOutOfRangeException is thrown.
+
+- Publisher-related members
+
+  Two read-only properties, `Publisher` and `Type`, return the value of the private `pubName` and `pubType` fields. The values are originally supplied by the call to the `Publication` class constructor.
+
+- Publishing-related members
+
+  Two methods, `Publish` and `GetPublicationDate`, set and return the publication date. The `Publish` method sets a private  `published` flag to `true` when it is called and assigns the date passed to it as an argument to the private `datePublished` field. The `GetPublicationDate` method returns the string "NYP" if the `published` flag is `false`, and the value of the `datePublished` field if it is `true`.
+
+- Copyright-related members
+
+  The `Copyright` method takes the name of the copyright holder and the year of the copyright as arguments and assigns them to the private `copyrName` and `copyrDate` fields. The values can be retrieved from the `CopyrightName` and `CopyrightDate` properties.
+
+- An override of the `ToString` method
+
+  If @System.Object.ToString is inherited, it returns the fully qualified name of the type, which is of little use in differentiating one instance from another. The `Publication` class overrides @System.Object.ToString to return the value of the `Title` property.
```

Comment:
The System.Object.ToString method is always inherited. I'd rewrite the first phrase as "If a type does not override the System.Object.ToString method, "